Gemfile.lock

```diff
@@ -616,69 +619,68 @@ GEM
     sawyer (0.9.2)
       addressable (>= 2.3.5)
       faraday (>= 0.17.3, < 3)
-    sentry-rails (5.4.2)
+    sentry-rails (5.16.1)
       railties (>= 5.0)
-      sentry-ruby (~> 5.4.2)
-    sentry-ruby (5.4.2)
+      sentry-ruby (~> 5.16.1)
+    sentry-ruby (5.16.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
     sentry-user_informer (0.2.0)
       rack
       railties (>= 6.1.0, < 7.1.0)
       sentry-rails (~> 5.4)
-    sexp_processor (4.12.1)
-    single_cov (1.3.2)
-    slop (3.6.0)
+    sexp_processor (4.17.1)
+    single_cov (1.5.0)
     snaky_hash (2.0.1)
       hashie
       version_gem (~> 1.1, >= 1.1.1)
     socksify (1.7.1)
-    soft_deletion (1.6.0)
-      activerecord (>= 4.2.0, < 6.2.0)
+    soft_deletion (1.9.0)
+      activerecord (>= 5.0.0, < 7.2)
     sprockets (3.7.2)
       concurrent-ruby (~> 1.0)
       rack (> 1, < 3)
     sprockets-rails (3.4.2)
       actionpack (>= 5.2)
       activesupport (>= 5.2)
       sprockets (>= 3.0.0)
-    sqlite3 (1.4.2)
+    sqlite3 (1.7.2)
+      mini_portile2 (~> 2.8.0)
     stackprof (0.2.12)
     terminal-table (1.8.0)
       unicode-display_width (~> 1.1, >= 1.1.1)
     thor (1.3.0)
-    tilt (2.0.10)
+    tilt (2.3.0)
     tzinfo (2.0.6)
       concurrent-ruby (~> 1.0)
     uglifier (3.2.0)
       execjs (>= 0.3.0, < 3)
-    unf (0.1.4)
-      unf_ext
-    unf_ext (0.0.8.2)
     unicode-display_width (1.8.0)
     validates_lengths_from_database (0.8.0)
       activerecord (>= 4)
     version_gem (1.1.3)
-    warden (1.2.7)
-      rack (>= 1.0)
-    webmock (3.0.1)
-      addressable (>= 2.3.6)
+    warden (1.2.9)
+      rack (>= 2.0.9)
+    webmock (3.20.0)
```

Comment:
### [webmock](https://github.com/bblimke/webmock)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.0.1 → 3.20.0

[_[change-log](https://github.com/bblimke/webmock/blob/v3.20.0/CHANGELOG.md), [source-code](https://github.com/bblimke/webmock/tree/v3.20.0), [gem-diff](https://my.diffend.io/gems/webmock/3.0.1/3.20.0)_]





<details>
<summary>Commits</summary>

A change of **442** commits. See the full changes on [the compare page](https://github.com/bblimke/webmock/compare/v3.0.1...v3.20.0).

These are the first 10 commits:
- (270e3f7) [Merge pull request #868 from ioquatix/patch-1](https://github.com/bblimke/webmock/commit/270e3f74c684b35e7ec488eff81560d3256314e4)
- (3054e73) [Merge pull request #871 from brandur/brandur-keyword-args](https://github.com/bblimke/webmock/commit/3054e732feb334ed764d3370b147208825103059)
- (05bdefc) [Work for getting this passing on ruby 2.7](https://github.com/bblimke/webmock/commit/05bdefc367484dab6652cf23347c7caef55bce04)
- (6682ad4) [Fix all the ruby 2.7 warnings coming from this code.](https://github.com/bblimke/webmock/commit/6682ad450d6d33235f8722781b286300620a92c0)
- (401f12e) [Version 3.8.0](https://github.com/bblimke/webmock/commit/401f12e39c2da6bb7e04fb4b8c1cded47ed14434)
- (a44eb00) [Upgraded version to 3.8.0](https://github.com/bblimke/webmock/commit/a44eb000b892f048b3158206993d2a78a08da61a)
- (bf418ec) [replace rbuf_fill modification with duplication of core libr…](https://github.com/bblimke/webmock/commit/bf418ec25424ed63899d9196546e05c5ba82ee70)
- (7a36d3c) [Merge pull request #873 from kipply/dont-tracepoint](https://github.com/bblimke/webmock/commit/7a36d3c6b67d0059f52e699f6e51738df11e1e70)
- (facc1a2) [Be able to mock non-ASCII bodies with HTTP.rb](https://github.com/bblimke/webmock/commit/facc1a21eb2ea2d9d8d7853b164f39acf8b17333)
- (895a7ec) [Merge pull request #874 from dentarg/http_rb-preserve-body-e…](https://github.com/bblimke/webmock/commit/895a7ec81c61c0ebadbbce66fd89bf0f90d1662a)


</details>